packages/simple-markdown/.editorconfig

```diff
@@ -0,0 +1,23 @@
+root = true
```

Comment:
I'd prefer not to have a customized editor-config here.  I also think `prettier` manages this all now so let's just delete this for now